sat/cli/bmccreds/parser.py

```diff
@@ -102,6 +122,29 @@ def add_bmccreds_subparser(subparsers):
              '--password.'
     )
 
+    bmccreds_subparser.add_argument(
+        '--password-length',
+        help='The desired length of the random BMC password. Only valid with '
+             '--random-password.'
+    )
+
+    bmccreds_subparser.add_argument(
+        '--password-char-sets',
+        help=f'A comma-separated list of allowed character sets to be used in '
+             f'a randomly generated password. Only valid with --random-password. '
+             f'Character sets are as follows: "alpha" includes all uppercase and '
+             f'lowercase ASCII letters, "numeric" includes digits 0-9, and "symbols" '
+             f'includes the symbols "{VALID_CHAR_SETS["symbols"].replace("%", "%%")}". ',
+        type=selected_chars, default='alpha,numeric,symbols'
+    )
+
+    bmccreds_subparser.add_argument(
+        '--password-chars',
```

Comment:
I would suggest making this mutually exclusive with `--password-char-sets`.  When neither is specified, we could default to the character sets alpha, numeric, and symbols.

Reply:
Addressed